pkg/cloud/services/networking/securitygroups.go

```diff
@@ -229,7 +241,7 @@ func matchGroups(desired, observed infrav1.SecurityGroup) bool {
 
 // reconcileGroup reconciles an already existing observed group by essentially emptying out all the rules and
```

Comment:
Nit: Instead of reconcileGroup it must be `reconcileGroupRules`

Reply:
@prankul88 fixed.